hax/treemakers/corrections.py

```diff
@@ -11,7 +11,8 @@ class Corrections(TreeMaker):
     Provides:
     - Corrected S1 contains xyz-correction:
       - cs1: The corrected area in pe of the main interaction's S1 using NN 3D FDC
-      - cs1_tpf_2dfdc: Same but for TPF 2D FDC
+      - cs1_true: Same but for after correction of electric field effects
```

Comment:
If this is what we should all finally use in the analysis, should we call it ```cs1``` instead? (Since previous ```cs1``` is not yet fully corrected, and so we don't need to change the variable in everyone's downstream analyses and ```lax```.)

Reply:
Can we keep two so that we can compare? Just in case one has some problem, for eg introducing some anomalous leakage?

Reply:
Even though the current cs1 is not the correct one to use, I agree that we could still keep it for comparison (I see how it can be confusing that the variable 'cs1' is not the fully the corrected one, but we can go step by step and update this variable once that the results support the new correction. As for the name of the variable, Dan suggested before to go from 'cs1_true' to 'cs1_field_corrected', both sounds good to me :)

Reply:
If we are confident enough (and I think we are from the plots you posted), I think we can directly name the new corrected S1 as "cs1" so all analyses will automatically take the correction, and copy the previous cs1 to "cs1_nofieldcorr" (or something).

Reply:
@pdeperio Good argument, agreed!

Reply:
Agree. I'll proceed and get the map with the extra suggestions you made and then make the PL for the new maps and also changing the name.